runtests.sh

```diff
@@ -19,7 +19,7 @@ function start_gemini_service {
 start_gemini_service
 
 export PYTHONPATH=`pwd`/src
-
+pip3 install virtualenv
```

Comment:
I think this should go into [cico_setup.sh:prep()](https://github.com/fabric8-analytics/fabric8-gemini-server/pull/1/files#diff-66536a55e28a60c7e24d2d4a43a8b89cR17)

Reply:
done.